cmd/hub/cli/cli.go

```diff
@@ -55,18 +64,25 @@ func Init(rootCmd *cobra.Command) {
 		config.Flags["session"].DefValue.(string),
 		"User session token")
 
-	if err := cmd.BindFlags(config.Viper, rootCmd, config.Flags); err != nil {
-		cmd.Fatal(err)
-	}
+	rootCmd.PersistentFlags().StringP(
```

Comment:
Oh, important config change... `org` has been moved to a global flag and config var on `hub`, and is no longer part of the `buck` config. This makes logical sense because `buckd` has no concern for orgs, but also make handling orgs much easier (hub can add the org context itself).